device-catalog.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheey" href="device-catalog.css">
+        <title>Интернет магазин Device. Каталог товаров</title>
+    </head>
+    <body>
+        <header>
+            <div>
+                <a class="nav_link" href=".//index.html"><img src=".//img/logo-black.png" alt="Logo device"></a>
```

Comment:
this logo can be just a text Device inside of anchor tag), but your version is not a mistake

Reply:
and what?) ok, let it be image, but it can be just text inside of link